geetools/ee_image_collection.py

```diff
@@ -1740,3 +1740,129 @@ def to_int(d):
         ax = plot_data("doy", data, "Day of Year", colors, ax)
 
         return ax
+
+    def reduceRegion(
+        self,
+        reducer: str,
+        geometry: ee.Geometry,
+        idProperty: str = "system:index",
+        idPropertyType: type = ee.Number,
+        propertyReducer: str | ee.Reducer = "first",
+        dateFormat: str | ee.String = EE_DATE_FORMAT,
+        numberFormat: str | ee.String = "%s",
```

Comment:
since we only pass one `idProperty` why not setting only one formatting param? could be something like `idFormat` or `idPropertyFormat`. This way we can reduce the amount of params.

Reply:
solved in https://github.com/gee-community/geetools/pull/359/commits/51df04895217d571122bdd778e6f7f3616b80914